### PR TITLE
feat: Bulk Management reorganization, phase 1

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -**
Re-organize the Bulk Management controls, remove tabs, convert Bulk Management tab to Bulk Management History View, and add an import success toast.

JIRA: [AU-43](https://openedx.atlassian.net/browse/AU-43)
figma: [figma] (https://www.figma.com/file/LHr3WTCMBe7GOm6MqXZwpk/Grading-Enhancements?node-id=329%3A125)

**What changed?**
Updated redux logic to:
* track and control active UI view, between grades and activity log
* Add trigger for showing an import success toast
Move "Tabs" to "Views"
* GradesTab => GradesView
* BulkManagementTab => BulkManagementHistoryView
Simplify BulkManagementHistoryView
* remove download button
* replace hints with help text (with new responsive behavior from Kevin)
* update header
Group BulkManagementControls
* BulkManagementControls now contains import and download grades buttons
* (20px margin between)
Other small updates to make this easier
* Add new InterventionsReport component to wrap the interventions download button and help text
* Split up Search Controls to remove FilterToggleButton (for cleaner view organization)
* Move messages to per-component files (was getting messy)
* Update margins around step display
* Rename UserLabel to FilteredUsersLabel and move it on the view
* updated test plan

**Developer Checklist**
- [x] Test suites passing
- [x] Documentation and test plan updated, if applicable
- [x] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Testing Instructions** (updates made to test plan)
* Navigate between views (verify both tables still load and the help texts are updated)
* verify import grades works and brings up the Toast (and that the toast dismisses after 5 secs)
* Verify Download buttons (grades and interventions) still work
* smoke test

image
image
image

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by jansenk** [comment]
😱  
Screen Shot 2021-07-30 at 11 26 11 AM

**Review by nsprenkle — APPROVED** [review]
Solid refactor! Behavior looks good outside of 2 open questions:
1. Should navigating to the Bulk Grade History view change the URL to allow for bookmarking?
2. Can we better raise the error when a bad Bulk Grade upload happens? There's a success toast but the error banner only shows in the Bulk Grade history page. Maybe just moving the banner to the main tab?

File: src/components/GradesView/GradesView.scss [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    background-color: #999;
+    opacity: 0.5;
+    z-index: 99999;
+    display:flex;
+    align-items: flex-start;
+    justify-content: center;
+    padding: 200px;
+}
+
+.color-black {
```
[the remark this answers is not in the document]
Comment (muselesscreator) [reply src/components/GradesView/GradesView.scss:16]: I believe this is just copied over?

File: src/components/NetworkButton/index.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+import selectors from 'data/selectors';
+import { StrictDict } from 'utils';
+
+export const buttonStates = StrictDict({
+  pending: 'pending',
+  default: 'default',
+});
+
+export class NetworkButton extends React.Component {
```
[the remark this answers is not in the document]
Comment (muselesscreator) [reply src/components/NetworkButton/index.jsx:18]: abs fair.

File: src/data/constants/app.js [hunk 3]
```diff
@@ -3,6 +3,11 @@ import { getConfig } from '@edx/frontend-platform';
 
 export const routePath = `${getConfig().PUBLIC_PATH}:courseId`;
 
+export const views = StrictDict({
+  grades: 'grades',
+  activity: 'activity',
```
[the remark this answers is not in the document]
Comment (muselesscreator) [reply src/data/constants/app.js:8]: yeeeah, I was thinking that as well.  Will modify

**Review by nsprenkle — APPROVED** [review]
Updates look good :shipit:

**Comment by jansenk** [comment]
Looks good to me, but we should set up a team smoke test n stage after this goes up

**Review by jansenk — APPROVED** [review]
